feat(feedback): Add UF screenshot capability (#9867)
getsentry/sentry-docs: docs/platforms/javascript/common/user-feedback/configuration/index.mdx

| ----------------- | ------------------------ | ------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
| `showName`        | `boolean`                | `true`                                | Displays the name field on the Feedback form, however will still capture the name (if available) from Sentry SDK context.   |
| `showEmail`       | `boolean`                | `true`                                | Displays the email field on the Feedback form, however will still capture the email (if available) from Sentry SDK context. |

> | `showScreenshot`  | `boolean`                | `true`                                | Allows the user to send a screenshot attachment with their feedback. Requires SDK v8. |

Reviewer: ```suggestion
| `showScreenshot`  | `boolean`                | `true`                                | Allows the user to send a screenshot attachment with their feedback. Requires SDK v8+. |
```

> | `showScreenshot`  | `boolean`                | `true`                                | Allows the user to send a screenshot attachment with their feedback. Requires SDK v8+. |

Reviewer 2: ```suggestion
| `showScreenshot`  | `boolean`                | `true`                                | Allows the user to send a screenshot attachment with their feedback. Requires SDK v8 and above. |
```

> | `showScreenshot`  | `boolean`                | `true`                                | Allows the user to send a screenshot attachment with their feedback. Requires SDK version 8.0.0 and above. For on-prem, release 24.4.2 is also required. |

Reviewer 3: ```suggestion
| `showScreenshot`  | `boolean`                | `true`                                | Allows the user to send a screenshot attachment with their feedback. Requires SDK version 8.0.0 and above. For self-hosted, release 24.4.2 is also required. |
```
